src/partition_func.jl

```diff
@@ -1,5 +1,33 @@
 using Interpolations
 
+"""
+Returns a Dict holding the default (Barklem & Collet 2016) partition functions.
+"""
+function setup_partition_funcs(atoms=joinpath(_data_dir, "BarklemCollet2016-atomic_partition.dat"),
+                              mols=joinpath(_data_dir, "BarklemCollet2016-molecular_partition.dat"))
+    atomUs, molUs = read_partition_funcs.([atoms, mols])
+    new_molUs = typeof(molUs)()
+    #use only neutral molecules, add "I"
+    for (m, f) in molUs
+        if !('+' in m) && !('-' in m)
```

Comment:
Yes, definitely.  I'm leaving it off for now because as far as I can tell, theres not charged particles in Vald.  That could definitely be a misunderstanding on my part, though.  Also, it would potentially require some more complicated code in the moleq system (although this depends how Barklem and Collet defined their equilibrium coefs).